OfficeDevPnP.Core/OfficeDevPnP.Core/Framework/Provisioning/Model/ComposedLook.cs

```diff
@@ -1,105 +1,91 @@
 ﻿using System;
+using OfficeDevPnP.Core.Framework.Provisioning.Model.Comparers;
+using OfficeDevPnP.Core.Framework.Provisioning.Model.HashFormatters;
+using OfficeDevPnP.Core.Framework.Provisioning.Model.Attributes;
 
 namespace OfficeDevPnP.Core.Framework.Provisioning.Model
 {
     /// <summary>
     /// Domain Object that defines a Composed Look in the Provision Template
     /// </summary>
-    public partial class ComposedLook : IEquatable<ComposedLook>
+    public partial class ComposedLook : ModelBase<ComposedLook>
     {
-        static ComposedLook()
+        public static ComposedLook Empty
         {
-            Empty = new ComposedLook();
+            get { return ComposedLookBuilder.Build(); }
         }
+        private class ComposedLookBuilder
```

Comment:
Ok this might be a bit overkill. But it's worth thinking about what kind of patterns to apply